sycl/source/backend/level_zero.cpp

```diff
@@ -0,0 +1,96 @@
+//==--------- level_zero.cpp - SYCL Level-Zero backend ---------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <CL/sycl.hpp>
+#include <detail/platform_impl.hpp>
+#include <detail/plugin.hpp>
+#include <detail/program_impl.hpp>
+#include <detail/queue_impl.hpp>
+
+__SYCL_INLINE_NAMESPACE(cl) {
+namespace sycl {
+namespace level0 {
+using namespace detail;
+
+// Get the L0 plugin.
```

Comment:
please re-use 
`template <backend BE> const plugin &pi::getPlugin();`

Reply:
Thx, fixed,